.cicdtemplate/.bitrise/bitrise.yml

```diff
@@ -0,0 +1,214 @@
+---
+format_version: '8'
+default_step_lib_source: 'https://github.com/bitrise-io/bitrise-steplib.git'
+project_type: android
+trigger_map:
+  - push_branch: develop
+    workflow: develop
+
+  - push_branch: release/*
+    workflow: release
+
+  - push_branch: master
+    workflow: master
+
+  - push_branch: '*'
+    workflow: primary
+
+  - pull_request_source_branch: '*'
+    workflow: primary
+
+# Work-flows definition
+workflows:
+  # Workflow on * branch
+  primary:
+    steps:
+      - activate-ssh-key@4:
+          run_if: '{{getenv "SSH_RSA_PRIVATE_KEY" | ne ""}}'
+
+      - git-clone@4: {}
+
+      - cache-pull@2: {}
+
+      - install-missing-android-tools@2:
+          inputs:
+            - gradlew_path: "$PROJECT_LOCATION/gradlew"
+
+      # Execute Lint check
+      - android-lint@0:
+          inputs:
+            - project_location: "$PROJECT_LOCATION"
+
+      # Execute Detekt
+      - gradle-runner@1:
+          inputs:
+            - gradlew_path: "$PROJECT_LOCATION/gradlew"
+            - gradle_task: detekt
+          title: Detekt
+
+      # Execute UnitTest
+      - android-unit-test@1:
+          inputs:
+            - arguments: jacocoTestReport
+            - variant: "$VARIANT"
+            - module: "$MODULE"
+            - report_path_pattern: '*build/reports'
+            - project_location: "$PROJECT_LOCATION"
+
+      # Upload jacocoTestReport to Apps & Artifacts for later downloading
+      - deploy-to-bitrise-io@1:
+          inputs:
+            - deploy_path: "$PROJECT_LOCATION/$MODULE/build/reports/jacoco/jacocoTestReport/html"
```

Comment:
Bitrise doesn't allow to archive folder, could you please check again?

Reply:
Ops... I forgot that point, we faced on TW project.